MetaMask/metamask-extension
# Changed revokePermissions to return `null` instead of an error for no permissions

## **Description**
This PR Changes `revokePermissions` to return `null` instead of an error when permissions exist for an origin.

## **Related issues**

Related: https://github.com/MetaMask/MetaMask-planning/issues/1681

## **Manual testing steps**

1. Go to this page https://docs.metamask.io/wallet/reference/wallet_requestpermissions/
2. Hit `Send Request` to request permissions
3. Click Next, Click Connect
4. Go to this page https://docs.metamask.io/wallet/reference/wallet_getpermissions/
5. Click `Send Request` to get the permissions
6. See that there is permissions
7. paste this in the console:
```js
await window.ethereum.request({
  "method": "wallet_revokePermissions",
  "params": [
    {
      "eth_accounts": {}
    }
  ]
});
```
8. Click `Send Request` to get the permissions
9. see that there is no permissions and they have been revoked.
10. try and revoke the permissions again
11. you should not get an error back from the provider, just `null`

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
